chemprop/data/utils.py

```diff
@@ -50,8 +55,8 @@ def preprocess_smiles_columns(path: str,
         else:
             smiles_columns = [None]*number_of_molecules
     else:
-        if not isinstance(smiles_columns,list):
-            smiles_columns=[smiles_columns]
+        if not isinstance(smiles_columns, list):
+            smiles_columns = [smiles_columns]
```

Comment:
what about the case where a `Sequence[str]` is passed? that fails this check and then is assumed to be of type `str`. Type-checking at runtime is very tenuous in python, so it's best practice to check on the most _restrictive_ type, which in this case is: `if isinstance(smiles_columns,str)`